pkg/proxycert/proxy.crt

```diff
@@ -0,0 +1,26 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Please can we generate the cert rather than hardcoding it

Reply:
this is a certificate file not hardcoded string